joomla/joomla-cms
# [4.0] Add the possibility to have a custom login logo

This PR Adds an Option in the Backend Template to add a custom Login Logo and to show the Sitename or not.

Sitename + Login Logo

![grafik](https://user-images.githubusercontent.com/828371/45357466-9bfcfb00-b5c6-11e8-9b9c-624643fd85b3.png)

Only Login Logo:
![grafik](https://user-images.githubusercontent.com/828371/45357509-bcc55080-b5c6-11e8-9dce-a1c921ee1c87.png)

None of both:
![grafik](https://user-images.githubusercontent.com/828371/45357542-d49cd480-b5c6-11e8-9ece-cfb54c976496.png)

Test Instructions:
1) Apply the patch
2) Change your desired settings Logo / Sitename

![grafik](https://user-images.githubusercontent.com/828371/45357587-f302d000-b5c6-11e8-8a64-748c11298adc.png)

3) logout of the backend
4) Login page should show your desired logo and or Sitename
5) Try with different combinations
